hotel/site_sections/hotel_assignments.py

```diff
@@ -138,6 +138,23 @@ def ordered(self, out, session):
             for a in group:
                 writerow(a, a.hotel_requests)
 
```

Comment:
btw, minor naming thing:  this could be renamed as 'Passkey' export format, as this is the system used by both the mark center and the Gaylord. http://passkey.com/  a bunch of other hotels use this as their backend too.

on a cooler note, this -should- work out of the box for Gaylord too.